waffle/locale/en_US/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,206 @@
+# SOME DESCRIPTIVE TITLE.
```

Comment:
Should these be filled out?

Reply:
Agreed, IMO it's OK to leave it as-is. 
Here's the same line from DRF, using Transifex as well: https://github.com/encode/django-rest-framework/blob/d2994e0596c3163ac970b29dad6a61485f938045/rest_framework/locale/en/LC_MESSAGES/django.po#L1